### DISCUSSION TITLE
Is there a setting available to display an alphabetical list of scripts in the "Choose SQL script" window?

### DISCUSSION THREAD

**Discussion by scokna4gh** [dbeaver/dbeaver#20116] [Q&A] [answered]
I am new to DBeaver and have searched for an answer before posting my question here. When I choose "Open SQL Script" the "Choose SQL script" window defaults to a chronological sort order. How do I default it to an alphabetical sort on the script name?

![image](https://github.com/dbeaver/dbeaver/assets/134745719/8b2523ba-13e7-42c3-8152-7f8e52c80deb)

**Comment by E1izabeth** [comment]
Have you tried clicking on the table header (Script)?

**Reply by scokna4gh** [reply]
Hi @E1izabeth,
Thanks for your reply. I click on the column header "Script" each time I open the list of scripts to sort the list of script names alphabetically. I am looking for a way to default the list to be sorted alphabetically so I don't have to click on the column header "Script" each time I open the script list.

Kind regards,
Scott

**Reply by E1izabeth** [reply, marked as answer]
There is no such setting
